internal/config/authentication.go

```diff
@@ -362,6 +371,22 @@ func (a *AuthenticationMethod[C]) validate() error {
 	return a.Method.validate()
 }
 
+type AuthenticationMethodCloudConfig struct {
+}
+
+func (a AuthenticationMethodCloudConfig) setDefaults(map[string]any) {}
+
+// info describes properties of the authentication method "cloud".
+func (a AuthenticationMethodCloudConfig) info() AuthenticationMethodInfo {
+	return AuthenticationMethodInfo{
+		Method:            auth.Method_METHOD_JWT,
```

Comment:
Might be worth making a new method. This way if you have both enable then you can discern that in audit logs.